/work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:283: FutureWarning: `torch.cuda.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cuda', args...)` instead.
  @autocast(enabled=False)
/work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:418: FutureWarning: `torch.cuda.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cuda', args...)` instead.
  @autocast(enabled=False)
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/torch/_dynamo/variables/functions.py:663: UserWarning: Graph break due to unsupported builtin torch._C._to_dlpack. This function is either a Python builtin (e.g. _warnings.warn) or a third-party C/C++ Python extension (perhaps created with pybind). If it is a Python builtin, please file an issue on GitHub so the PyTorch team can add support for it and see the next case for a workaround. If it is a third-party C/C++ Python extension, please either wrap it into a PyTorch-understood custom operator (see https://pytorch.org/tutorials/advanced/custom_ops_landing_page.html for more details) or, if it is traceable, use torch.compiler.allow_in_graph.
  torch._dynamo.utils.warn_once(msg)
V0421 04:17:59.200745 47393259730176 torch/_dynamo/guards.py:2611] [4/1] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0421 04:17:59.200745 47393259730176 torch/_dynamo/guards.py:2611] [4/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:17:59.200745 47393259730176 torch/_dynamo/guards.py:2611] [4/1] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 04:17:59.215682 47393259730176 torch/_dynamo/guards.py:2611] [5/1] [__recompiles] Recompiling function to_dgl_nd in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/__init__.py:141
V0421 04:17:59.215682 47393259730176 torch/_dynamo/guards.py:2611] [5/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:17:59.215682 47393259730176 torch/_dynamo/guards.py:2611] [5/1] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 04:17:59.224410 47393259730176 torch/_dynamo/guards.py:2611] [6/1] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0421 04:17:59.224410 47393259730176 torch/_dynamo/guards.py:2611] [6/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:17:59.224410 47393259730176 torch/_dynamo/guards.py:2611] [6/1] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 04:17:59.257366 47393259730176 torch/_dynamo/guards.py:2611] [8/1] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0421 04:17:59.257366 47393259730176 torch/_dynamo/guards.py:2611] [8/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:17:59.257366 47393259730176 torch/_dynamo/guards.py:2611] [8/1] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
W0421 04:18:06.100231 47393259730176 torch/_dynamo/variables/tensor.py:715] [30/0] Graph break from `Tensor.item()`, consider setting:
W0421 04:18:06.100231 47393259730176 torch/_dynamo/variables/tensor.py:715] [30/0]     torch._dynamo.config.capture_scalar_outputs = True
W0421 04:18:06.100231 47393259730176 torch/_dynamo/variables/tensor.py:715] [30/0] or:
W0421 04:18:06.100231 47393259730176 torch/_dynamo/variables/tensor.py:715] [30/0]     env TORCHDYNAMO_CAPTURE_SCALAR_OUTPUTS=1
W0421 04:18:06.100231 47393259730176 torch/_dynamo/variables/tensor.py:715] [30/0] to include these operations in the captured graph.
W0421 04:18:06.100231 47393259730176 torch/_dynamo/variables/tensor.py:715] [30/0] 
V0421 04:18:09.174035 47393259730176 torch/_dynamo/guards.py:2611] [4/2] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0421 04:18:09.174035 47393259730176 torch/_dynamo/guards.py:2611] [4/2] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:09.174035 47393259730176 torch/_dynamo/guards.py:2611] [4/2] [__recompiles]     - tensor 'L['row']' size mismatch at index 0. expected 0, actual 3883
V0421 04:18:09.174035 47393259730176 torch/_dynamo/guards.py:2611] [4/2] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 04:18:09.202264 47393259730176 torch/_dynamo/guards.py:2611] [5/2] [__recompiles] Recompiling function to_dgl_nd in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/__init__.py:141
V0421 04:18:09.202264 47393259730176 torch/_dynamo/guards.py:2611] [5/2] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:09.202264 47393259730176 torch/_dynamo/guards.py:2611] [5/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 0, actual 3883
V0421 04:18:09.202264 47393259730176 torch/_dynamo/guards.py:2611] [5/2] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 04:18:09.214102 47393259730176 torch/_dynamo/guards.py:2611] [6/2] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0421 04:18:09.214102 47393259730176 torch/_dynamo/guards.py:2611] [6/2] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:09.214102 47393259730176 torch/_dynamo/guards.py:2611] [6/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 0, actual 3883
V0421 04:18:09.214102 47393259730176 torch/_dynamo/guards.py:2611] [6/2] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 04:18:09.252531 47393259730176 torch/_dynamo/guards.py:2611] [8/2] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0421 04:18:09.252531 47393259730176 torch/_dynamo/guards.py:2611] [8/2] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:09.252531 47393259730176 torch/_dynamo/guards.py:2611] [8/2] [__recompiles]     - tensor 'L['col']' size mismatch at index 0. expected 0, actual 3883
V0421 04:18:09.252531 47393259730176 torch/_dynamo/guards.py:2611] [8/2] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 04:18:09.269847 47393259730176 torch/_dynamo/guards.py:2611] [32/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_574 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:574
V0421 04:18:09.269847 47393259730176 torch/_dynamo/guards.py:2611] [32/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:09.269847 47393259730176 torch/_dynamo/guards.py:2611] [32/1] [__recompiles]     - L['___stack0'] == 2579                                      
V0421 04:18:09.488425 47393259730176 torch/_dynamo/guards.py:2611] [34/1] [__recompiles] Recompiling function cat in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:222
V0421 04:18:09.488425 47393259730176 torch/_dynamo/guards.py:2611] [34/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:09.488425 47393259730176 torch/_dynamo/guards.py:2611] [34/1] [__recompiles]     - tensor 'L['seq'][0]' size mismatch at index 0. expected 0, actual 3883
V0421 04:18:10.269825 47393259730176 torch/_dynamo/guards.py:2611] [20/1] [__recompiles] Recompiling function add_edges in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:435
V0421 04:18:10.269825 47393259730176 torch/_dynamo/guards.py:2611] [20/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:10.269825 47393259730176 torch/_dynamo/guards.py:2611] [20/1] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 3883, actual 780
V0421 04:18:10.291836 47393259730176 torch/_dynamo/guards.py:2611] [26/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_541 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:541
V0421 04:18:10.291836 47393259730176 torch/_dynamo/guards.py:2611] [26/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:10.291836 47393259730176 torch/_dynamo/guards.py:2611] [26/1] [__recompiles]     - tensor 'L['v']' size mismatch at index 0. expected 3883, actual 780
V0421 04:18:10.320336 47393259730176 torch/_dynamo/guards.py:2611] [27/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_542 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:542
V0421 04:18:10.320336 47393259730176 torch/_dynamo/guards.py:2611] [27/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:10.320336 47393259730176 torch/_dynamo/guards.py:2611] [27/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 3883, actual 780
V0421 04:18:10.336889 47393259730176 torch/_dynamo/guards.py:2611] [28/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_545 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:545
V0421 04:18:10.336889 47393259730176 torch/_dynamo/guards.py:2611] [28/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:10.336889 47393259730176 torch/_dynamo/guards.py:2611] [28/1] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 3883, actual 780
V0421 04:18:10.404266 47393259730176 torch/_dynamo/guards.py:2611] [30/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_573 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:573
V0421 04:18:10.404266 47393259730176 torch/_dynamo/guards.py:2611] [30/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:10.404266 47393259730176 torch/_dynamo/guards.py:2611] [30/1] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 3883, actual 780
V0421 04:18:10.664386 47393259730176 torch/_dynamo/guards.py:2611] [32/2] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_574 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:574
V0421 04:18:10.664386 47393259730176 torch/_dynamo/guards.py:2611] [32/2] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:10.664386 47393259730176 torch/_dynamo/guards.py:2611] [32/2] [__recompiles]     - tensor 'L['v']' size mismatch at index 0. expected 3883, actual 780
V0421 04:18:10.664386 47393259730176 torch/_dynamo/guards.py:2611] [32/2] [__recompiles]     - L['___stack0'] == 2579                                      
V0421 04:18:10.916896 47393259730176 torch/_dynamo/guards.py:2611] [34/2] [__recompiles] Recompiling function cat in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:222
V0421 04:18:10.916896 47393259730176 torch/_dynamo/guards.py:2611] [34/2] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:10.916896 47393259730176 torch/_dynamo/guards.py:2611] [34/2] [__recompiles]     - tensor 'L['seq'][1]' size mismatch at index 0. expected 3883, actual 780
V0421 04:18:10.916896 47393259730176 torch/_dynamo/guards.py:2611] [34/2] [__recompiles]     - tensor 'L['seq'][0]' size mismatch at index 0. expected 0, actual 7766
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/torch/_dynamo/variables/functions.py:663: UserWarning: Graph break due to unsupported builtin posix.stat. This function is either a Python builtin (e.g. _warnings.warn) or a third-party C/C++ Python extension (perhaps created with pybind). If it is a Python builtin, please file an issue on GitHub so the PyTorch team can add support for it and see the next case for a workaround. If it is a third-party C/C++ Python extension, please either wrap it into a PyTorch-understood custom operator (see https://pytorch.org/tutorials/advanced/custom_ops_landing_page.html for more details) or, if it is traceable, use torch.compiler.allow_in_graph.
  torch._dynamo.utils.warn_once(msg)
V0421 04:18:15.105578 47393259730176 torch/_dynamo/guards.py:2611] [6/3] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0421 04:18:15.105578 47393259730176 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:15.105578 47393259730176 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 04:18:15.105578 47393259730176 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 04:18:15.105578 47393259730176 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 04:18:15.163527 47393259730176 torch/_dynamo/guards.py:2611] [7/1] [__recompiles] Recompiling function torch_dynamo_resume_in_zerocopy_to_dgl_ndarray_at_435 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:435
V0421 04:18:15.163527 47393259730176 torch/_dynamo/guards.py:2611] [7/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:15.163527 47393259730176 torch/_dynamo/guards.py:2611] [7/1] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/torch/_dynamo/variables/functions.py:663: UserWarning: Graph break due to unsupported builtin time.perf_counter. This function is either a Python builtin (e.g. _warnings.warn) or a third-party C/C++ Python extension (perhaps created with pybind). If it is a Python builtin, please file an issue on GitHub so the PyTorch team can add support for it and see the next case for a workaround. If it is a third-party C/C++ Python extension, please either wrap it into a PyTorch-understood custom operator (see https://pytorch.org/tutorials/advanced/custom_ops_landing_page.html for more details) or, if it is traceable, use torch.compiler.allow_in_graph.
  torch._dynamo.utils.warn_once(msg)
V0421 04:18:15.703317 47393259730176 torch/_dynamo/guards.py:2611] [86/1] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 04:18:15.703317 47393259730176 torch/_dynamo/guards.py:2611] [86/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:15.703317 47393259730176 torch/_dynamo/guards.py:2611] [86/1] [__recompiles]     - L['___stack0'] == 930537.175410288                          
V0421 04:18:15.800415 47393259730176 torch/_dynamo/guards.py:2611] [86/2] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 04:18:15.800415 47393259730176 torch/_dynamo/guards.py:2611] [86/2] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:15.800415 47393259730176 torch/_dynamo/guards.py:2611] [86/2] [__recompiles]     - L['___stack0'] == 930537.188214345                          
V0421 04:18:15.800415 47393259730176 torch/_dynamo/guards.py:2611] [86/2] [__recompiles]     - L['___stack0'] == 930537.175410288                          
V0421 04:18:18.994163 47393259730176 torch/_dynamo/guards.py:2611] [86/3] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 04:18:18.994163 47393259730176 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:18.994163 47393259730176 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     - L['___stack0'] == 930537.285260552                          
V0421 04:18:18.994163 47393259730176 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     - L['___stack0'] == 930537.188214345                          
V0421 04:18:18.994163 47393259730176 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     - L['___stack0'] == 930537.175410288                          
V0421 04:18:20.752464 47393259730176 torch/_dynamo/guards.py:2611] [86/4] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 04:18:20.752464 47393259730176 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:20.752464 47393259730176 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 930540.478849736                          
V0421 04:18:20.752464 47393259730176 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 930537.285260552                          
V0421 04:18:20.752464 47393259730176 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 930537.188214345                          
V0421 04:18:20.752464 47393259730176 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 930537.175410288                          
V0421 04:18:20.820562 47393259730176 torch/_dynamo/guards.py:2611] [86/5] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 04:18:20.820562 47393259730176 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:20.820562 47393259730176 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 930542.237218153                          
V0421 04:18:20.820562 47393259730176 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 930540.478849736                          
V0421 04:18:20.820562 47393259730176 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 930537.285260552                          
V0421 04:18:20.820562 47393259730176 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 930537.188214345                          
V0421 04:18:20.820562 47393259730176 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 930537.175410288                          
V0421 04:18:20.826725 47393259730176 torch/_dynamo/guards.py:2611] [86/6] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 04:18:20.826725 47393259730176 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:20.826725 47393259730176 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 930542.305398633                          
V0421 04:18:20.826725 47393259730176 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 930542.237218153                          
V0421 04:18:20.826725 47393259730176 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 930540.478849736                          
V0421 04:18:20.826725 47393259730176 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 930537.285260552                          
V0421 04:18:20.826725 47393259730176 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 930537.188214345                          
V0421 04:18:20.826725 47393259730176 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 930537.175410288                          
V0421 04:18:20.883557 47393259730176 torch/_dynamo/guards.py:2611] [57/1] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/view.py:82
V0421 04:18:20.883557 47393259730176 torch/_dynamo/guards.py:2611] [57/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:20.883557 47393259730176 torch/_dynamo/guards.py:2611] [57/1] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=0
V0421 04:18:20.905756 47393259730176 torch/_dynamo/guards.py:2611] [61/1] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:690
V0421 04:18:20.905756 47393259730176 torch/_dynamo/guards.py:2611] [61/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:20.905756 47393259730176 torch/_dynamo/guards.py:2611] [61/1] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 04:18:20.925311 47393259730176 torch/_dynamo/guards.py:2611] [62/1] [__recompiles] Recompiling function update_column in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:772
V0421 04:18:20.925311 47393259730176 torch/_dynamo/guards.py:2611] [62/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:20.925311 47393259730176 torch/_dynamo/guards.py:2611] [62/1] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 04:18:20.945169 47393259730176 torch/_dynamo/guards.py:2611] [63/1] [__recompiles] Recompiling function create in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:472
V0421 04:18:20.945169 47393259730176 torch/_dynamo/guards.py:2611] [63/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:20.945169 47393259730176 torch/_dynamo/guards.py:2611] [63/1] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 04:18:20.962924 47393259730176 torch/_dynamo/guards.py:2611] [64/1] [__recompiles] Recompiling function __init__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:210
V0421 04:18:20.962924 47393259730176 torch/_dynamo/guards.py:2611] [64/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:20.962924 47393259730176 torch/_dynamo/guards.py:2611] [64/1] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=0
V0421 04:18:20.982607 47393259730176 torch/_dynamo/guards.py:2611] [65/1] [__recompiles] Recompiling function infer_scheme in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:151
V0421 04:18:20.982607 47393259730176 torch/_dynamo/guards.py:2611] [65/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:20.982607 47393259730176 torch/_dynamo/guards.py:2611] [65/1] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=0
V0421 04:18:21.029156 47393259730176 torch/_dynamo/guards.py:2611] [68/1] [__recompiles] Recompiling function func in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:213
V0421 04:18:21.029156 47393259730176 torch/_dynamo/guards.py:2611] [68/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:21.029156 47393259730176 torch/_dynamo/guards.py:2611] [68/1] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=0
V0421 04:18:21.355427 47393259730176 torch/_dynamo/guards.py:2611] [70/1] [__recompiles] Recompiling function torch_dynamo_resume_in_gspmm_at_75 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:75
V0421 04:18:21.355427 47393259730176 torch/_dynamo/guards.py:2611] [70/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:21.355427 47393259730176 torch/_dynamo/guards.py:2611] [70/1] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=0
V0421 04:18:21.373651 47393259730176 torch/_dynamo/guards.py:2611] [74/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:163
V0421 04:18:21.373651 47393259730176 torch/_dynamo/guards.py:2611] [74/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:21.373651 47393259730176 torch/_dynamo/guards.py:2611] [74/1] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
V0421 04:18:21.390041 47393259730176 torch/_dynamo/guards.py:2611] [77/1] [__recompiles] Recompiling function zeros in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:287
V0421 04:18:21.390041 47393259730176 torch/_dynamo/guards.py:2611] [77/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:21.390041 47393259730176 torch/_dynamo/guards.py:2611] [77/1] [__recompiles]     - L['shape'][1] == 745                                        
V0421 04:18:21.663455 47393259730176 torch/_dynamo/guards.py:2611] [6/4] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0421 04:18:21.663455 47393259730176 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:21.663455 47393259730176 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 04:18:21.663455 47393259730176 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 04:18:21.663455 47393259730176 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 04:18:21.663455 47393259730176 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 04:18:21.719041 47393259730176 torch/_dynamo/guards.py:2611] [79/1] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray_for_write in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:454
V0421 04:18:21.719041 47393259730176 torch/_dynamo/guards.py:2611] [79/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:21.719041 47393259730176 torch/_dynamo/guards.py:2611] [79/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 745, actual 128
V0421 04:18:21.756325 47393259730176 torch/_dynamo/guards.py:2611] [80/1] [__recompiles] Recompiling function check_is_view in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:441
V0421 04:18:21.756325 47393259730176 torch/_dynamo/guards.py:2611] [80/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:21.756325 47393259730176 torch/_dynamo/guards.py:2611] [80/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 745, actual 128
V0421 04:18:21.766672 47393259730176 torch/_dynamo/guards.py:2611] [81/1] [__recompiles] Recompiling function torch_dynamo_resume_in_check_is_view_at_443 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:443
V0421 04:18:21.766672 47393259730176 torch/_dynamo/guards.py:2611] [81/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:21.766672 47393259730176 torch/_dynamo/guards.py:2611] [81/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 745, actual 128
V0421 04:18:21.778309 47393259730176 torch/_dynamo/guards.py:2611] [82/1] [__recompiles] Recompiling function torch_dynamo_resume_in_zerocopy_to_dgl_ndarray_for_write_at_461 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:461
V0421 04:18:21.778309 47393259730176 torch/_dynamo/guards.py:2611] [82/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:21.778309 47393259730176 torch/_dynamo/guards.py:2611] [82/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 745, actual 128
V0421 04:18:21.792151 47393259730176 torch/_dynamo/guards.py:2611] [83/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_165 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:165
V0421 04:18:21.792151 47393259730176 torch/_dynamo/guards.py:2611] [83/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:21.792151 47393259730176 torch/_dynamo/guards.py:2611] [83/1] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
V0421 04:18:21.837019 47393259730176 torch/_dynamo/guards.py:2611] [86/7] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 04:18:21.837019 47393259730176 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:21.837019 47393259730176 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 930542.311569462                          
V0421 04:18:21.837019 47393259730176 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 930542.305398633                          
V0421 04:18:21.837019 47393259730176 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 930542.237218153                          
V0421 04:18:21.837019 47393259730176 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 930540.478849736                          
V0421 04:18:21.837019 47393259730176 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 930537.285260552                          
V0421 04:18:21.837019 47393259730176 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 930537.188214345                          
V0421 04:18:21.837019 47393259730176 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 930537.175410288                          
V0421 04:18:21.843378 47393259730176 torch/_dynamo/guards.py:2611] [86/8] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 04:18:21.843378 47393259730176 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:21.843378 47393259730176 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 930543.321808707                          
V0421 04:18:21.843378 47393259730176 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 930542.311569462                          
V0421 04:18:21.843378 47393259730176 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 930542.305398633                          
V0421 04:18:21.843378 47393259730176 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 930542.237218153                          
V0421 04:18:21.843378 47393259730176 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 930540.478849736                          
V0421 04:18:21.843378 47393259730176 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 930537.285260552                          
V0421 04:18:21.843378 47393259730176 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 930537.188214345                          
V0421 04:18:21.843378 47393259730176 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 930537.175410288                          
W0421 04:18:21.843517 47393259730176 torch/_dynamo/convert_frame.py:762] [86/8] torch._dynamo hit config.cache_size_limit (8)
W0421 04:18:21.843517 47393259730176 torch/_dynamo/convert_frame.py:762] [86/8]    function: 'torch_dynamo_resume_in_hook_at_12' (/work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12)
W0421 04:18:21.843517 47393259730176 torch/_dynamo/convert_frame.py:762] [86/8]    last reason: L['___stack0'] == 930537.175410288                          
W0421 04:18:21.843517 47393259730176 torch/_dynamo/convert_frame.py:762] [86/8] To log all recompilation reasons, use TORCH_LOGS="recompiles".
W0421 04:18:21.843517 47393259730176 torch/_dynamo/convert_frame.py:762] [86/8] To diagnose recompilation issues, see https://pytorch.org/docs/main/torch.compiler_troubleshooting.html.
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 04:18:23.038246 47393259730176 torch/_dynamo/guards.py:2611] [0/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:449
V0421 04:18:23.038246 47393259730176 torch/_dynamo/guards.py:2611] [0/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:23.038246 47393259730176 torch/_dynamo/guards.py:2611] [0/1] [__recompiles]     - tensor 'L['feats']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:23.065082 47393259730176 torch/_dynamo/guards.py:2611] [1/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:154
V0421 04:18:23.065082 47393259730176 torch/_dynamo/guards.py:2611] [1/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:23.065082 47393259730176 torch/_dynamo/guards.py:2611] [1/1] [__recompiles]     - tensor 'L['feats']' size mismatch at index 0. expected 2580, actual 2611
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 04:18:23.081231 47393259730176 torch/_dynamo/guards.py:2611] [9/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_157 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:157
V0421 04:18:23.081231 47393259730176 torch/_dynamo/guards.py:2611] [9/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:23.081231 47393259730176 torch/_dynamo/guards.py:2611] [9/1] [__recompiles]     - tensor 'L['h']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:23.097571 47393259730176 torch/_dynamo/guards.py:2611] [13/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_157 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:157
V0421 04:18:23.097571 47393259730176 torch/_dynamo/guards.py:2611] [13/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:23.097571 47393259730176 torch/_dynamo/guards.py:2611] [13/1] [__recompiles]     - tensor 'L['h']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:23.138713 47393259730176 torch/_dynamo/guards.py:2611] [4/3] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0421 04:18:23.138713 47393259730176 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:23.138713 47393259730176 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 04:18:23.138713 47393259730176 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     - L['num_dst'] == 2580                                        
V0421 04:18:23.138713 47393259730176 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     - L['num_dst'] == 2580                                        
V0421 04:18:23.170220 47393259730176 torch/_dynamo/guards.py:2611] [8/3] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0421 04:18:23.170220 47393259730176 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:23.170220 47393259730176 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 04:18:23.170220 47393259730176 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     - L['___stack2'] == 2580                                      
V0421 04:18:23.170220 47393259730176 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     - L['___stack2'] == 2580                                      
V0421 04:18:23.184010 47393259730176 torch/_dynamo/guards.py:2611] [15/1] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 04:18:23.184010 47393259730176 torch/_dynamo/guards.py:2611] [15/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:23.184010 47393259730176 torch/_dynamo/guards.py:2611] [15/1] [__recompiles]     - L['data'][0] == 2580                                        
V0421 04:18:26.409750 47393259730176 torch/_dynamo/guards.py:2611] [29/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_572 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:572
V0421 04:18:26.409750 47393259730176 torch/_dynamo/guards.py:2611] [29/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:26.409750 47393259730176 torch/_dynamo/guards.py:2611] [29/1] [__recompiles]     - L['___stack0'] == 2580                                      
V0421 04:18:26.427699 47393259730176 torch/_dynamo/guards.py:2611] [30/2] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_573 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:573
V0421 04:18:26.427699 47393259730176 torch/_dynamo/guards.py:2611] [30/2] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:26.427699 47393259730176 torch/_dynamo/guards.py:2611] [30/2] [__recompiles]     - L['___stack0'] == 2580                                      
V0421 04:18:26.427699 47393259730176 torch/_dynamo/guards.py:2611] [30/2] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 3883, actual 3876
V0421 04:18:26.696526 47393259730176 torch/_dynamo/guards.py:2611] [34/3] [__recompiles] Recompiling function cat in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:222
V0421 04:18:26.696526 47393259730176 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:26.696526 47393259730176 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     - 2 <= L['seq'][0].size()[0] <= 2147483645                      # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 04:18:26.696526 47393259730176 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     - tensor 'L['seq'][1]' size mismatch at index 0. expected 3883, actual 3876
V0421 04:18:26.696526 47393259730176 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     - tensor 'L['seq'][1]' size mismatch at index 0. expected 3883, actual 3876
V0421 04:18:26.936649 47393259730176 torch/_dynamo/guards.py:2611] [4/4] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0421 04:18:26.936649 47393259730176 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:26.936649 47393259730176 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - tensor 'L['row']' size mismatch at index 0. expected 0, actual 3876
V0421 04:18:26.936649 47393259730176 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 04:18:26.936649 47393259730176 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - L['num_dst'] == 2580                                        
V0421 04:18:26.936649 47393259730176 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - tensor 'L['row']' size mismatch at index 0. expected 0, actual 3876
V0421 04:18:26.975962 47393259730176 torch/_dynamo/guards.py:2611] [8/4] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0421 04:18:26.975962 47393259730176 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:26.975962 47393259730176 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - tensor 'L['col']' size mismatch at index 0. expected 0, actual 3876
V0421 04:18:26.975962 47393259730176 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 04:18:26.975962 47393259730176 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - L['___stack2'] == 2580                                      
V0421 04:18:26.975962 47393259730176 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - tensor 'L['col']' size mismatch at index 0. expected 0, actual 3876
V0421 04:18:26.997044 47393259730176 torch/_dynamo/guards.py:2611] [41/1] [__recompiles] Recompiling function spmatrix in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/sparse/sparse_matrix.py:763
V0421 04:18:26.997044 47393259730176 torch/_dynamo/guards.py:2611] [41/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:26.997044 47393259730176 torch/_dynamo/guards.py:2611] [41/1] [__recompiles]     - tensor 'L['indices']' stride mismatch at index 0. expected 9326, actual 9312
V0421 04:18:27.285224 47393259730176 torch/_dynamo/guards.py:2611] [47/1] [__recompiles] Recompiling function arange in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:406
V0421 04:18:27.285224 47393259730176 torch/_dynamo/guards.py:2611] [47/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:27.285224 47393259730176 torch/_dynamo/guards.py:2611] [47/1] [__recompiles]     - L['stop'] == 2580                                           
V0421 04:18:27.516013 47393259730176 torch/_dynamo/guards.py:2611] [48/1] [__recompiles] Recompiling function in_degrees in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:702
V0421 04:18:27.516013 47393259730176 torch/_dynamo/guards.py:2611] [48/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:27.516013 47393259730176 torch/_dynamo/guards.py:2611] [48/1] [__recompiles]     - tensor 'L['v']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:27.534705 47393259730176 torch/_dynamo/guards.py:2611] [50/1] [__recompiles] Recompiling function has_nodes in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2878
V0421 04:18:27.534705 47393259730176 torch/_dynamo/guards.py:2611] [50/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:27.534705 47393259730176 torch/_dynamo/guards.py:2611] [50/1] [__recompiles]     - tensor 'L['vid']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:27.551441 47393259730176 torch/_dynamo/guards.py:2611] [51/1] [__recompiles] Recompiling function torch_dynamo_resume_in_has_nodes_at_2925 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2925
V0421 04:18:27.551441 47393259730176 torch/_dynamo/guards.py:2611] [51/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:27.551441 47393259730176 torch/_dynamo/guards.py:2611] [51/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:27.791025 47393259730176 torch/_dynamo/guards.py:2611] [52/1] [__recompiles] Recompiling function torch_dynamo_resume_in_has_nodes_at_2926 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2926
V0421 04:18:27.791025 47393259730176 torch/_dynamo/guards.py:2611] [52/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:27.791025 47393259730176 torch/_dynamo/guards.py:2611] [52/1] [__recompiles]     - tensor 'L['vid_tensor']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:27.814793 47393259730176 torch/_dynamo/guards.py:2611] [53/1] [__recompiles] Recompiling function has_nodes in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:444
V0421 04:18:27.814793 47393259730176 torch/_dynamo/guards.py:2611] [53/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:27.814793 47393259730176 torch/_dynamo/guards.py:2611] [53/1] [__recompiles]     - tensor 'L['vids']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:27.832015 47393259730176 torch/_dynamo/guards.py:2611] [54/1] [__recompiles] Recompiling function torch_dynamo_resume_in_has_nodes_at_2930 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2930
V0421 04:18:27.832015 47393259730176 torch/_dynamo/guards.py:2611] [54/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:27.832015 47393259730176 torch/_dynamo/guards.py:2611] [54/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:28.060605 47393259730176 torch/_dynamo/guards.py:2611] [55/1] [__recompiles] Recompiling function sum in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:152
V0421 04:18:28.060605 47393259730176 torch/_dynamo/guards.py:2611] [55/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:28.060605 47393259730176 torch/_dynamo/guards.py:2611] [55/1] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:28.142610 47393259730176 torch/_dynamo/guards.py:2611] [57/2] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/view.py:82
V0421 04:18:28.142610 47393259730176 torch/_dynamo/guards.py:2611] [57/2] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:28.142610 47393259730176 torch/_dynamo/guards.py:2611] [57/2] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=1
V0421 04:18:28.142610 47393259730176 torch/_dynamo/guards.py:2611] [57/2] [__recompiles]     - tensor 'L['val']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:28.161175 47393259730176 torch/_dynamo/guards.py:2611] [61/2] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:690
V0421 04:18:28.161175 47393259730176 torch/_dynamo/guards.py:2611] [61/2] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:28.161175 47393259730176 torch/_dynamo/guards.py:2611] [61/2] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=1
V0421 04:18:28.161175 47393259730176 torch/_dynamo/guards.py:2611] [61/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:28.180185 47393259730176 torch/_dynamo/guards.py:2611] [62/2] [__recompiles] Recompiling function update_column in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:772
V0421 04:18:28.180185 47393259730176 torch/_dynamo/guards.py:2611] [62/2] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:28.180185 47393259730176 torch/_dynamo/guards.py:2611] [62/2] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=1
V0421 04:18:28.180185 47393259730176 torch/_dynamo/guards.py:2611] [62/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:28.199698 47393259730176 torch/_dynamo/guards.py:2611] [63/2] [__recompiles] Recompiling function create in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:472
V0421 04:18:28.199698 47393259730176 torch/_dynamo/guards.py:2611] [63/2] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:28.199698 47393259730176 torch/_dynamo/guards.py:2611] [63/2] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=1
V0421 04:18:28.199698 47393259730176 torch/_dynamo/guards.py:2611] [63/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:28.217048 47393259730176 torch/_dynamo/guards.py:2611] [64/2] [__recompiles] Recompiling function __init__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:210
V0421 04:18:28.217048 47393259730176 torch/_dynamo/guards.py:2611] [64/2] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:28.217048 47393259730176 torch/_dynamo/guards.py:2611] [64/2] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=1
V0421 04:18:28.217048 47393259730176 torch/_dynamo/guards.py:2611] [64/2] [__recompiles]     - tensor 'L['storage']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:28.236469 47393259730176 torch/_dynamo/guards.py:2611] [65/2] [__recompiles] Recompiling function infer_scheme in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:151
V0421 04:18:28.236469 47393259730176 torch/_dynamo/guards.py:2611] [65/2] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:28.236469 47393259730176 torch/_dynamo/guards.py:2611] [65/2] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=1
V0421 04:18:28.236469 47393259730176 torch/_dynamo/guards.py:2611] [65/2] [__recompiles]     - tensor 'L['tensor']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:28.281260 47393259730176 torch/_dynamo/guards.py:2611] [68/2] [__recompiles] Recompiling function func in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:213
V0421 04:18:28.281260 47393259730176 torch/_dynamo/guards.py:2611] [68/2] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:28.281260 47393259730176 torch/_dynamo/guards.py:2611] [68/2] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=1
V0421 04:18:28.281260 47393259730176 torch/_dynamo/guards.py:2611] [68/2] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:28.295121 47393259730176 torch/_dynamo/guards.py:2611] [70/2] [__recompiles] Recompiling function torch_dynamo_resume_in_gspmm_at_75 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:75
V0421 04:18:28.295121 47393259730176 torch/_dynamo/guards.py:2611] [70/2] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:28.295121 47393259730176 torch/_dynamo/guards.py:2611] [70/2] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=1
V0421 04:18:28.295121 47393259730176 torch/_dynamo/guards.py:2611] [70/2] [__recompiles]     - tensor 'L['lhs_data']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:28.313361 47393259730176 torch/_dynamo/guards.py:2611] [74/2] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:163
V0421 04:18:28.313361 47393259730176 torch/_dynamo/guards.py:2611] [74/2] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:28.313361 47393259730176 torch/_dynamo/guards.py:2611] [74/2] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=1
V0421 04:18:28.313361 47393259730176 torch/_dynamo/guards.py:2611] [74/2] [__recompiles]     - tensor 'L['X']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:28.329957 47393259730176 torch/_dynamo/guards.py:2611] [77/2] [__recompiles] Recompiling function zeros in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:287
V0421 04:18:28.329957 47393259730176 torch/_dynamo/guards.py:2611] [77/2] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:28.329957 47393259730176 torch/_dynamo/guards.py:2611] [77/2] [__recompiles]     - L['shape'][0] == 2580                                       
V0421 04:18:28.329957 47393259730176 torch/_dynamo/guards.py:2611] [77/2] [__recompiles]     - L['shape'][0] == 2580                                       
V0421 04:18:28.501502 47393259730176 torch/_dynamo/guards.py:2611] [79/2] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray_for_write in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:454
V0421 04:18:28.501502 47393259730176 torch/_dynamo/guards.py:2611] [79/2] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:28.501502 47393259730176 torch/_dynamo/guards.py:2611] [79/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:28.501502 47393259730176 torch/_dynamo/guards.py:2611] [79/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:28.544041 47393259730176 torch/_dynamo/guards.py:2611] [80/2] [__recompiles] Recompiling function check_is_view in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:441
V0421 04:18:28.544041 47393259730176 torch/_dynamo/guards.py:2611] [80/2] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:28.544041 47393259730176 torch/_dynamo/guards.py:2611] [80/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:28.544041 47393259730176 torch/_dynamo/guards.py:2611] [80/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:28.556358 47393259730176 torch/_dynamo/guards.py:2611] [81/2] [__recompiles] Recompiling function torch_dynamo_resume_in_check_is_view_at_443 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:443
V0421 04:18:28.556358 47393259730176 torch/_dynamo/guards.py:2611] [81/2] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:28.556358 47393259730176 torch/_dynamo/guards.py:2611] [81/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:28.556358 47393259730176 torch/_dynamo/guards.py:2611] [81/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:28.569787 47393259730176 torch/_dynamo/guards.py:2611] [82/2] [__recompiles] Recompiling function torch_dynamo_resume_in_zerocopy_to_dgl_ndarray_for_write_at_461 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:461
V0421 04:18:28.569787 47393259730176 torch/_dynamo/guards.py:2611] [82/2] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:28.569787 47393259730176 torch/_dynamo/guards.py:2611] [82/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:28.569787 47393259730176 torch/_dynamo/guards.py:2611] [82/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:28.587146 47393259730176 torch/_dynamo/guards.py:2611] [88/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:569
V0421 04:18:28.587146 47393259730176 torch/_dynamo/guards.py:2611] [88/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:28.587146 47393259730176 torch/_dynamo/guards.py:2611] [88/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:28.682893 47393259730176 torch/_dynamo/guards.py:2611] [89/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_590 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:590
V0421 04:18:28.682893 47393259730176 torch/_dynamo/guards.py:2611] [89/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:28.682893 47393259730176 torch/_dynamo/guards.py:2611] [89/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:28.755997 47393259730176 torch/_dynamo/guards.py:2611] [90/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:418
V0421 04:18:28.755997 47393259730176 torch/_dynamo/guards.py:2611] [90/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:28.755997 47393259730176 torch/_dynamo/guards.py:2611] [90/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:29.501994 47393259730176 torch/_dynamo/guards.py:2611] [93/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_432 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:432
V0421 04:18:29.501994 47393259730176 torch/_dynamo/guards.py:2611] [93/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:29.501994 47393259730176 torch/_dynamo/guards.py:2611] [93/1] [__recompiles]     - L['shape'][1] == 2580                                       
V0421 04:18:30.849301 47393259730176 torch/_dynamo/guards.py:2611] [94/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_595 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:595
V0421 04:18:30.849301 47393259730176 torch/_dynamo/guards.py:2611] [94/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:30.849301 47393259730176 torch/_dynamo/guards.py:2611] [94/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:31.117620 47393259730176 torch/_dynamo/guards.py:2611] [95/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_604 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:604
V0421 04:18:31.117620 47393259730176 torch/_dynamo/guards.py:2611] [95/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:31.117620 47393259730176 torch/_dynamo/guards.py:2611] [95/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:31.875751 47393259730176 torch/_dynamo/guards.py:2611] [96/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_648 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:648
V0421 04:18:31.875751 47393259730176 torch/_dynamo/guards.py:2611] [96/1] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:31.875751 47393259730176 torch/_dynamo/guards.py:2611] [96/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:31.986499 47393259730176 torch/_dynamo/guards.py:2611] [57/3] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/view.py:82
V0421 04:18:31.986499 47393259730176 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:31.986499 47393259730176 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=0
V0421 04:18:31.986499 47393259730176 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     - tensor 'L['val']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:31.986499 47393259730176 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=0
V0421 04:18:32.007528 47393259730176 torch/_dynamo/guards.py:2611] [61/3] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:690
V0421 04:18:32.007528 47393259730176 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:32.007528 47393259730176 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 04:18:32.007528 47393259730176 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:32.007528 47393259730176 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 04:18:32.028700 47393259730176 torch/_dynamo/guards.py:2611] [62/3] [__recompiles] Recompiling function update_column in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:772
V0421 04:18:32.028700 47393259730176 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:32.028700 47393259730176 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 04:18:32.028700 47393259730176 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:32.028700 47393259730176 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 04:18:32.050241 47393259730176 torch/_dynamo/guards.py:2611] [63/3] [__recompiles] Recompiling function create in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:472
V0421 04:18:32.050241 47393259730176 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:32.050241 47393259730176 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 04:18:32.050241 47393259730176 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:32.050241 47393259730176 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 04:18:32.069902 47393259730176 torch/_dynamo/guards.py:2611] [64/3] [__recompiles] Recompiling function __init__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:210
V0421 04:18:32.069902 47393259730176 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:32.069902 47393259730176 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=0
V0421 04:18:32.069902 47393259730176 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     - tensor 'L['storage']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:32.069902 47393259730176 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=0
V0421 04:18:32.091407 47393259730176 torch/_dynamo/guards.py:2611] [65/3] [__recompiles] Recompiling function infer_scheme in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:151
V0421 04:18:32.091407 47393259730176 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:32.091407 47393259730176 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=0
V0421 04:18:32.091407 47393259730176 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     - tensor 'L['tensor']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:32.091407 47393259730176 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=0
V0421 04:18:32.141076 47393259730176 torch/_dynamo/guards.py:2611] [68/3] [__recompiles] Recompiling function func in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:213
V0421 04:18:32.141076 47393259730176 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:32.141076 47393259730176 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=0
V0421 04:18:32.141076 47393259730176 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:32.141076 47393259730176 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=0
V0421 04:18:32.156016 47393259730176 torch/_dynamo/guards.py:2611] [70/3] [__recompiles] Recompiling function torch_dynamo_resume_in_gspmm_at_75 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:75
V0421 04:18:32.156016 47393259730176 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:32.156016 47393259730176 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=0
V0421 04:18:32.156016 47393259730176 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     - tensor 'L['lhs_data']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:32.156016 47393259730176 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=0
V0421 04:18:32.175088 47393259730176 torch/_dynamo/guards.py:2611] [74/3] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:163
V0421 04:18:32.175088 47393259730176 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:32.175088 47393259730176 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
V0421 04:18:32.175088 47393259730176 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     - tensor 'L['X']' size mismatch at index 0. expected 2580, actual 2611
V0421 04:18:32.175088 47393259730176 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 04:18:32.591067 47393259730176 torch/_dynamo/guards.py:2611] [15/2] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 04:18:32.591067 47393259730176 torch/_dynamo/guards.py:2611] [15/2] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:32.591067 47393259730176 torch/_dynamo/guards.py:2611] [15/2] [__recompiles]     - L['data'][0] == 2611                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 04:18:32.591067 47393259730176 torch/_dynamo/guards.py:2611] [15/2] [__recompiles]     - L['data'][0] == 2580                                        
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 04:18:36.274601 47393259730176 torch/_dynamo/guards.py:2611] [15/3] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 04:18:36.274601 47393259730176 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:36.274601 47393259730176 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     - L['data'][0] == 2692                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 04:18:36.274601 47393259730176 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     - L['data'][0] == 2611                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 04:18:36.274601 47393259730176 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     - L['data'][0] == 2580                                        
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 04:18:40.143721 47393259730176 torch/_dynamo/guards.py:2611] [15/4] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 04:18:40.143721 47393259730176 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:40.143721 47393259730176 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 2615                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 04:18:40.143721 47393259730176 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 2692                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 04:18:40.143721 47393259730176 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 2611                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 04:18:40.143721 47393259730176 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 2580                                        
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 04:18:43.906354 47393259730176 torch/_dynamo/guards.py:2611] [15/5] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 04:18:43.906354 47393259730176 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:43.906354 47393259730176 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 2584                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 04:18:43.906354 47393259730176 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 2615                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 04:18:43.906354 47393259730176 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 2692                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 04:18:43.906354 47393259730176 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 2611                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 04:18:43.906354 47393259730176 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 2580                                        
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 04:18:47.591721 47393259730176 torch/_dynamo/guards.py:2611] [15/6] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 04:18:47.591721 47393259730176 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:47.591721 47393259730176 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 2631                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 04:18:47.591721 47393259730176 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 2584                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 04:18:47.591721 47393259730176 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 2615                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 04:18:47.591721 47393259730176 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 2692                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 04:18:47.591721 47393259730176 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 2611                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 04:18:47.591721 47393259730176 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 2580                                        
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 04:18:51.339042 47393259730176 torch/_dynamo/guards.py:2611] [15/7] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 04:18:51.339042 47393259730176 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:51.339042 47393259730176 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 2573                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 04:18:51.339042 47393259730176 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 2631                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 04:18:51.339042 47393259730176 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 2584                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 04:18:51.339042 47393259730176 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 2615                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 04:18:51.339042 47393259730176 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 2692                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 04:18:51.339042 47393259730176 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 2611                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 04:18:51.339042 47393259730176 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 2580                                        
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 04:18:54.977511 47393259730176 torch/_dynamo/guards.py:2611] [15/8] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 04:18:54.977511 47393259730176 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     triggered by the following guard failure(s):
V0421 04:18:54.977511 47393259730176 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 2620                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 04:18:54.977511 47393259730176 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 2573                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 04:18:54.977511 47393259730176 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 2631                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 04:18:54.977511 47393259730176 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 2584                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 04:18:54.977511 47393259730176 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 2615                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 04:18:54.977511 47393259730176 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 2692                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 04:18:54.977511 47393259730176 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 2611                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 04:18:54.977511 47393259730176 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 2580                                        
W0421 04:18:54.977700 47393259730176 torch/_dynamo/convert_frame.py:762] [15/8] torch._dynamo hit config.cache_size_limit (8)
W0421 04:18:54.977700 47393259730176 torch/_dynamo/convert_frame.py:762] [15/8]    function: '_dispatch' (/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65)
W0421 04:18:54.977700 47393259730176 torch/_dynamo/convert_frame.py:762] [15/8]    last reason: L['data'][0] == 2580                                        
W0421 04:18:54.977700 47393259730176 torch/_dynamo/convert_frame.py:762] [15/8] To log all recompilation reasons, use TORCH_LOGS="recompiles".
W0421 04:18:54.977700 47393259730176 torch/_dynamo/convert_frame.py:762] [15/8] To diagnose recompilation issues, see https://pytorch.org/docs/main/torch.compiler_troubleshooting.html.
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
Compiling model with torch.compile()...
Saved sparsity snapshot to outputs_compiled/tran/a-photo/SAGE/seed_2/sparsity_before.csv
                               layer_name  num_elements   sparsity
0  _orig_mod.encoder.graph_layer_1.weight        555025    0.00000
1    _orig_mod.encoder.graph_layer_1.bias           745  100.00000
2  _orig_mod.encoder.graph_layer_2.weight         95360    0.00000
3    _orig_mod.encoder.graph_layer_2.bias           128  100.00000
4      _orig_mod.encoder.decoder_1.weight        555025    0.00000
5        _orig_mod.encoder.decoder_1.bias           745    0.00000
6      _orig_mod.encoder.decoder_2.weight        555025    0.00018
7        _orig_mod.encoder.decoder_2.bias           745    0.00000
8         _orig_mod.encoder.linear.weight          1024    0.00000
9           _orig_mod.encoder.linear.bias             8    0.00000
Saved sparsity snapshot to outputs_compiled/tran/a-photo/SAGE/seed_2/sparsity_after.csv
                               layer_name  num_elements   sparsity
0  _orig_mod.encoder.graph_layer_1.weight        555025   8.085402
1    _orig_mod.encoder.graph_layer_1.bias           745   0.268456
2  _orig_mod.encoder.graph_layer_2.weight         95360  51.725042
3    _orig_mod.encoder.graph_layer_2.bias           128   0.000000
4      _orig_mod.encoder.decoder_1.weight        555025  68.287194
5        _orig_mod.encoder.decoder_1.bias           745   0.000000
6      _orig_mod.encoder.decoder_2.weight        555025  91.108328
7        _orig_mod.encoder.decoder_2.bias           745  63.489933
8         _orig_mod.encoder.linear.weight          1024   8.496094
9           _orig_mod.encoder.linear.bias             8   0.000000
Saved forward timing report to outputs_compiled/tran/a-photo/SAGE/seed_2/forward_times.csv
                         layer_name  avg_forward_time_ms  num_calls
0         _orig_mod.encoder.dropout             0.000934       8015
1   _orig_mod.encoder.graph_layer_1             0.001244       8015
2   _orig_mod.encoder.graph_layer_2             0.001021       8015
3       _orig_mod.encoder.decoder_1             0.000997       8015
4       _orig_mod.encoder.decoder_2             0.000991       8015
5          _orig_mod.encoder.linear             0.000340       8015
6   _orig_mod.encoder.vq.project_in             0.000960       8015
7  _orig_mod.encoder.vq.project_out             0.001081       8015
8    _orig_mod.encoder.vq._codebook             0.001189       8015
CPU Memory Usage: 1295.96 MB
Max GPU Memory Used: 1280.63 MB
